### PR TITLE
refactor(demo): update demo apps - EUBFR-69

### PR DESCRIPTION
**Pull request by raduchiriac**
Updates after change of projects API.

### CODE REVIEW

File: demo/budg/dashboard/src/pages/files/List.js [hunk 1]
```diff
@@ -48,28 +48,27 @@ class List extends Component {
...
     if (files.length === 0) {
       return (
         <div>
-          <button
-            className="ecl-button ecl-button--default"
-            onClick={this.loadFiles}
-          >
-            Refresh
-          </button>
+          {refreshButton}
```
Comment (kalinchernev) [inline demo/budg/dashboard/src/pages/files/List.js]: Let's try to keep jsx tags as close to html as possible for consistency:

```js
    const RefreshButton = () => (
      <button
        className="ecl-button ecl-button--default"
        onClick={this.loadFiles}
      >
        Refresh
      </button>
    );

    if (files.length === 0) {
      return (
        <div>
          <RefreshButton />
          <p className="ecl-paragraph">No file found</p>
        </div>
      );
    }

    return (
      <div className="files-list">
        <RefreshButton />
        <FilesList files={files} />
      </div>
    );
```

File: demo/website/src/App.js [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 import ProjectsList from './components/ProjectsList';
+import elasticsearch from 'elasticsearch';
 
 import './App.css';
 
 const App = () => (
   <div className="App">
     <h1>EUBFR Website</h1>
-    <ProjectsList />
+    <ProjectsList elasticsearch={elasticsearch} />
```
Comment (kalinchernev) [inline demo/website/src/App.js]: Is there a specific reason to pass an npm module as a prop? I think it's ok to include it in the `ProjectsList.js` as no obvious need to listen for rendering events on this type of object?

File: demo/website/src/components/ProjectsList.js [hunk 3]
```diff
@@ -23,29 +16,36 @@ class ProjectsList extends Component {
   }
 
   componentDidMount() {
+    const { elasticsearch } = this.props;
```
Comment (kalinchernev) [inline demo/website/src/components/ProjectsList.js]: I think the best place to get the dependency is in the top of the file as usual.
The client instantiation would make sense to be in the constructor function, no?
I don't think the client really needs to know anything about the DOM?

**Comment by raduchiriac** [comment]
I had to introduce a new rule in our `.eslintrc` because of how Elasticsearch formats the returned JSON

```yaml
no-underscore-dangle:
    - error
    - allow:
      - _source
```

File: .eslintignore [hunk 4]
```diff
@@ -1,2 +1,3 @@
 .serverless
 node_modules
+dist
```
Comment (kalinchernev) [inline .eslintignore]: Thanks for adding this, it was blocking the local linting task

**Pull request merged**